### PR TITLE
Task 14

### PR DESCRIPTION
**Pull request by varachkin**

### CODE REVIEW

File: task_14/js/task2.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+let c = 20;
```
Comment (Killerkip10) [inline task_14/js/task2.js:1]: Where is the second way?

File: task_14/js/task3.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+let day = prompt('введите количество дней');
+let second;
+if (day > 27 && day < 32){
```
Comment (Killerkip10) [inline task_14/js/task3.js:3]: Hmmm, I also want to know how many seconds in 2, 3, 10 and etc. days)

File: task_14/js/task4.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+let a = prompt('введите первое число');
+let e = prompt('введите операцию');
+let b = prompt('введите второе число');
+let result;
+a = +a;
+b = +b;
+if (e == '+'){
```
Comment (Killerkip10) [inline task_14/js/task4.js:7]: You can use if without else and your code will more readable
`
if (e == '-') {
   ...
}
if (e == '*') {
   ...
}
`

File: task_14/js/task4.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+let a = prompt('введите первое число');
+let e = prompt('введите операцию');
```
Comment (Killerkip10) [inline task_14/js/task4.js:2]: I think name for this variable should be `operation`

File: task_14/js/task5.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+let name = prompt('введите Ваше имя');
+if (name == 0 || name == ' '){
```
Comment (Killerkip10) [inline task_14/js/task5.js:2]: If I enter 5 spaces?

File: task_14/js/task7.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+let p = document.getElementById('id');
+let x = prompt('Введите число');
+let temp;
+let result = 1;
+while (x > 0){
+    temp = x % 10;
+    result = result * temp;
```
Comment (Killerkip10) [inline task_14/js/task7.js:7]: You can make this code less
result *= x % 10;

File: task_14/js/task2.js (hunk 1 above)
Comment (varachkin) [reply task_14/js/task2.js:1]: done

File: task_14/js/task3.js (hunk 2 above)
Comment (varachkin) [reply task_14/js/task3.js:3]: done

File: task_14/js/task4.js (hunk 3 above)
Comment (varachkin) [reply task_14/js/task4.js:7]: done

File: task_14/js/task4.js (hunk 4 above)
Comment (varachkin) [reply task_14/js/task4.js:2]: done

File: task_14/js/task5.js (hunk 5 above)
Comment (varachkin) [reply task_14/js/task5.js:2]: done

File: task_14/js/task7.js (hunk 6 above)
Comment (varachkin) [reply task_14/js/task7.js:7]: done

File: task_14/js/task3.js [hunk 7]
```diff
@@ -0,0 +1,3 @@
+let day = prompt('введите количество дней');
+let second = day * 86400;
```
Comment (Killerkip10) [inline task_14/js/task3.js:2]: I think will be better
let second = day * 60 * 60 * 24;

File: task_14/js/task5.js [hunk 8]
```diff
@@ -0,0 +1,7 @@
+let name = prompt('введите Ваше имя');
+let clearName = name.trim();
+if (clearName == 0 || clearName == ' '){
```
Comment (Killerkip10) [inline task_14/js/task5.js:3]: clearName == 0
Are you sure?)
I think 
`if (clearName == ''){` is more understandable

File: task_14/js/task6.js [hunk 9]
```diff
@@ -0,0 +1,8 @@
+let x = prompt('введите число');
+x = +x;
```
Comment (Killerkip10) [inline task_14/js/task6.js:2]: Redundant row

**Review by Killerkip10 — COMMENTED** [review]
## Task 15
### Total 100 points

**Pull request merged**